src/averageAges.js

```diff
@@ -37,7 +50,18 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const womens = people.filter((person) =>
```

Comment:
woman - one
women - multiple
```suggestion
  const women = people.filter((person) =>
```